internal/serialization/object_data_test.go

```diff
@@ -214,9 +203,9 @@ func TestObjectDataInput_ReadUTF2(t *testing.T) {
 	o := NewObjectDataOutput(0, nil, false)
 	o.WriteString("Furkan Şenharputlu")
 	o.WriteString("Jack")
-	o.WriteString("⚐中💦2😭‍🙆😔")
```

Comment:
nit: what is the reason for this change? Having multiple UTF-8 code point chars is better than a single one.

Reply:
staticcheck complains about using unicode code instead of some non-printable char. I'll add a few more chars.